docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
-    Consult the [Docker reference manual](https://docs.docker.com/engine/reference/commandline/run/) or run
-    `docker <command> --help` for more information on the docker command line interface.
-
-    * You can now run the host and client as we did for mock mode. The only differences are that the host must be run from
-     inside the container, and the host JAR will be named `host-simulation.jar` rather than `host-mock.jar`.
-    * Press `CTRL+D` to exit the container when finished.
-
-    On Windows systems, it is also possible to install the Windows subsystem for linux (2) and download ubuntu 20.04
-    from the Windows store. From there, it should be possible to launch an Ubuntu 20.04 shell from the start menu or
-    desktop and proceed to build or run Conclave applications as you would in a native Linux environment.
-    Instructions for installing wsl-2 can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install).
-    Please note that this method hasn't been extensively tested.
+2. Instantiate a container using the `conclave-build` image using
+   ```bash
+   docker run -it --rm -p 8080:8080 -v ${PWD}:/project -w /project conclave-build /bin/bash
+   ```
+   This will give you a bash shell in the container that simulates a native Linux machine. You can tweak this command
+   according to the specific needs of your project. Please take a look at the explanation of the options used in this
+   command [at the end of this tutorial](#appendix-summary-of-docker-command-options). Check the [Docker reference
+   manual](https://docs.docker.com/engine/reference/commandline/run/) or run `docker <command> --help` on the Docker
+   command line interface for more information.
+3. Run the host and client as you did for mock mode. The only differences are that the host must be run from inside
+   the container, and the host JAR is named `host-simulation.jar` rather than `host-mock.jar`.
+4. Press `CTRL+D` to exit the container when finished.
+
+Alternatively, you can also install the Windows subsystem for Linux (2) and download Ubuntu 20.04 from the Windows
+store. From there, you can launch an Ubuntu 20.04 shell from the start menu and build or run Conclave applications
+as in a native Linux environment.
+Instructions for installing WSL-2 can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install).
+Please note that this method hasn't been extensively tested.
 
 === "macOS"
-    A Linux environment is required to run the host in simulation mode. This section describes how to instantiate such an
-    environment using Conclave's Docker integration.
 
-    !!! note
-        * Docker is only required to run the *host* on macOS. The client can be run without Docker.
+You need a Linux environment to run the host in simulation mode. This section describes how to create such an
+environment using Conclave's Docker integration.
+
+    !!!Note
```

Comment:
For macOS we need to mention that Apple Silicon isn't supported

Reply:
Done.